src/wallet/wallet.ts

```diff
@@ -945,12 +1002,17 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    * @memberof HathorWalletServiceWallet
    * @inner
    */
-  async sendTransaction(address: string, value: number, options: { token?: string, changeAddress?: string, pinCode?: string } = {}): Promise<Transaction> {
+  async sendTransaction(
+    address: string,
+    value: number,
+    options: { token?: string; changeAddress?: string; pinCode?: string } = {}
+  ): Promise<Transaction> {
     this.failIfWalletNotReady();
-    const newOptions = Object.assign({
+    const newOptions = {
       token: '00',
-      changeAddress: null
-    }, options);
+      changeAddress: undefined,
```

Comment:
Passing a `changeAddress` as null was not expected by the `sendManyOutputsTransaction` method and had to be changed to `undefined` to comply with the expected type `changeAddress?: string;`. Otherwise the build phase would break.

The tests had to be changed accordingly. Since this is a static type issue, it's not expected that it will cause any issues with the wallet-lib consumers.

### Note
Another alternative solution would be adding `null` to the valid types for this parameter: `changeAddress?: string | null`.